### PR TITLE
Touchpad: add disable with mouse switch

### LINKED ISSUE #72: Add setting to disable touchpad when external mouse connected
Mentioned here:
https://www.reddit.com/r/elementaryos/comments/ao5fcz/add_a_feature_on_the_mouse_touchpad_settings/

Happy to implement this if this is a setting that we think should exist.

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/69775633-add-setting-to-disable-touchpad-when-external-mouse-connected?utm_campaign=plugin&utm_content=tracker%2F60165266&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F60165266&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by danrabbit**
Fixes #72

### CODE REVIEW

**Comment by danrabbit** [comment]
Requesting review from @cassidyjames on the new string

File: src/Views/Touchpad.vala [hunk 1]
```diff
@@ -136,6 +147,14 @@ public class MouseTouchpad.TouchpadView : Gtk.Grid {
             click_method_transform_func
         );
 
+        disable_with_mouse_switch.notify["active"].connect (() => {
+            if (disable_with_mouse_switch.active) {
+                glib_settings.set_enum ("send-events", 2);
```
Comment (davidmhewitt) [inline src/Views/Touchpad.vala]: Not a fan of these magic numbers for the enums. I was part way through  writing a branch when you opened this PR. I do some probably overly complicated binding transforms, so I guess you could either take code from here or maybe `set_string` works instead:
https://github.com/elementary/switchboard-plug-mouse-touchpad/commit/78adef2bde637f5ca097913899fcbb26ac3cc870

**Comment by cassidyjames** [comment]
It looks like macOS uses a checkbox and uses the term “ignore.” I think a switch still works here for our conventions, but is “Ignore” any better?

**Comment by cassidyjames** [comment]
It looks like Windows 10 goes the inverted route but still uses a checkbox (probably because it's not an explicit state being toggled on or off?)

Windows 10 Settings

**Comment by cassidyjames** [comment]
I guess we have “Disable while typing” right above this, so maybe your wording is fine…

**Comment by danrabbit** [comment]
@cassidyjames That's interesting that macOS distinguishes with 'built-in'. Can you confirm that your magic trackpad triggers this feature? Maybe we should adopt that distinction as well

**Comment by cassidyjames** [comment]
@danrabbit yeah I can check 👍

**Pull request merged**